### ISSUE TITLE
Ignore zero width spaces for buffer search

### ISSUE THREAD

**Issue by nyuszika7h** [weechat/weechat#946]
Zero width spaces should be skipped over when searching for text in a buffer. They're frequently used to prevent highlights.

**Comment by flashcode** [comment]
Hi,
For buffer search, you mean Ctrl+R?
And you want to ignore zero width spaces in messages? Can you give me an example of what you type, the content of message and what you should find?
Thanks.

**Comment by nyuszika7h** [comment]
Yes, I mean Ctrl+R. Since the raw log displays it unescaped as well (#947), here's a hexdump:

```
0000000: 3c41 e280 8d65 e280 8d72 e280 8d6f e280  <A...e...r...o..
0000010: 8d47 e280 8d47 3e20 7961 0a              .G...G> ya.
```

I'm searching for "AeroGG" (or any substring longer than 1 character) but it won't find it.
